packages/core/src/mf.ts

```diff
@@ -0,0 +1,49 @@
+import { createRsbuild, mergeRsbuildConfig } from '@rsbuild/core';
+import { composeCreateRsbuildConfig } from './config';
+
+import type { RsbuildConfig, RsbuildInstance } from '@rsbuild/core';
+import type { RslibConfig } from './types';
+
+export async function startMFDevServer(
```

Comment:
IIUC, after this is merged, we could remove https://github.com/rspack-contrib/storybook-rsbuild/blob/d574bc218836299aa07ca60b6bb3a90c6a01ec27/packages/addon-rslib/src/mf.ts logic?

Reply:
yes, i have done this, and will open a pr